manifests/m/MariaDB/Server/11.3.2/MariaDB.Server.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: MariaDB.Server
+PackageVersion: 11.3.2
+PackageLocale: en-US
+Publisher: MariaDB Corporation Ab
+PublisherUrl: https://mariadb.org
+PrivacyUrl: https://mariadb.org/privacy-policy/
+Author: MariaDB Corporation Ab
+PackageName: MariaDB
+PackageUrl: https://mariadb.org
+License: GPL 2.0 only
+LicenseUrl: https://mariadb.com/kb/en/mariadb-license/
+Copyright: Copyright (c) 2022 MariaDB. All rights reserved.
+ShortDescription: Community developed fork of MySQL server.
+Moniker: mariadb

```

```suggestion
Moniker: mariadb
ReleaseNotesUrl: https://mariadb.com/kb/en/mariadb-11-3-2-release-notes/
```

Since bot is complaining about it